### PR TITLE
compromises: log4j, js-faker-colors, golang-buildpacks, wp-apthemes

### PR DESCRIPTION
**Pull request by bureado**
Input welcome on a few references to recent cases. I also reviewed the snap CVEs but as far as I could read they weren't about snaps themselves breaking confinement but about users getting root via abusing the snap architecture.

### CODE REVIEW

**Comment by bureado** [comment]
cc @SantiagoTorres

**Comment by bureado** [comment]
Of note: https://www-computerweekly-com.cdn.ampproject.org/c/s/www.computerweekly.com/news/252512087/Russian-backed-hackers-defaced-Ukrainian-websites-as-cover-for-dangerous-malware-attack?amp=1

**Comment by PushkarJ** [comment]
Friendly TAG Security Tech Lead here :) 

@bureado thank you for adding these compromises. It sounds like we have a couple of things to fix before this can be merged:

1. Few spellings are [failing](https://github.com/cncf/tag-security/runs/5269457677?check_suite_focus=true) because spellchecker does not know these terms. If you add them to this list, spellchecker should be able to pass: https://github.com/cncf/tag-security/blob/main/ci/spelling-config.json#L8
1. There is a link for `rsa.com` that is throwing 404 [error](https://github.com/cncf/tag-security/runs/5269457626?check_suite_focus=true), if you can replace it with something else the link checker should pass too

**Comment by d-lanm** [comment]
There's a version of the URL that is 404ing on archive.org: https://web.archive.org/web/20180130205601/https://www.rsa.com/content/dam/pdfs/2-2017/kingslayer-a-supply-chain-attack.pdf

File: supply-chain-security/compromises/2022/wp-apthemes.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# WordPress theme publisher compromised, backdoor introduced in several products
+
```
Comment (TheFoxAtWork) [inline supply-chain-security/compromises/2022/wp-apthemes.md:2]:
```suggestion
<!-- cSpell:ignore webshell -->
```

File: supply-chain-security/compromises/2021/log4j.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Remote code injection in Log4j
+<!-- cSpell:ignore composability codeql cldrn -->
```
Comment (TheFoxAtWork) [inline supply-chain-security/compromises/2021/log4j.md:2]:
```suggestion
<!-- cSpell:ignore composability codeql cldrn hotpatch hotpatching -->
```

**Review by TheFoxAtWork — APPROVED** [review]

**Pull request merged**